.github/CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 <h3>New features since last release</h3>
 
+* The quantum state of a QNode can now be returned using the ``state()`` return function.
+  [(#789)](https://github.com/XanaduAI/pennylane/pull/789)
+  
+  Consider the following QNode:
+  ```python
+  dev = qml.device("default.qubit", wires=3)
+  
+  @qml.qnode(dev)
+  def qnode(x, y):
+      qml.RZ(x, wires=0)
+      qml.CNOT(wires=[0, 1])
+      qml.RY(y, wires=1)
+      qml.CNOT(wires=[0, 2])
+      return qml.state(range(3))
+  ```
+  
+  Calling the QNode will return its state
+  
+  ```pycon
+  >>> qnode(0.56, 0.1)
+  array([0.95985437-0.27601028j, 0.        +0.j        ,
+       0.04803275-0.01381203j, 0.        +0.j        ,
+       0.        +0.j        , 0.        +0.j        ,
+       0.        +0.j        , 0.        +0.j        ])
+  ```
+  
+  Differentiating the state is not yet fully supported, but is currently available when using the
```

Comment:
Linked to this comment: https://github.com/PennyLaneAI/pennylane/pull/737/files#r467349528
![image](https://user-images.githubusercontent.com/49409390/92801938-0eef0700-f384-11ea-9d8c-3533f34ec3cd.png)